core/controllers/learner_dashboard_test.py

```diff
@@ -301,10 +301,35 @@ def test_get_message_summaries(self):
         messages_summary = response_dict['message_summary_list']
 
         # Check the summary of the second message.
+        self.assertEqual(len(messages_summary), 2)
         second_message = messages_summary[1]
         self.assertDictContainsSubset({
             'text': 'Message 1',
             'author_username': 'editor'
         }, second_message)
 
         self.logout()
+
+    def test_anonymous_feedback_is_recorded_correctly(self):
```

Comment:
Can you make the test names a bit more descriptive following the guidelines [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests) - go through point 2 specifically.

Make the changes for all the test in the file if they don't follow the guideline.

Reply:
@bansalnitish this was the test name recommended to me earlier.

Reply:
Hm, @bansalnitish is right about following the guidelines, but in this case, I think the test name is reasonable -- the action is the submission of anonymous feedback and the outcome is that it's recorded correctly. 

So, l think I'd suggest that we go ahead and merge this if there are no other comments. @nithusha21 @bansalnitish if you agree please could you approve and merge?

Reply:
Okay, sounds good!